ONSdigital/dp-frontend-filter-dataset-controller
# Release/1.16.0

### What

- Updated `previewPage` vendor
- Exposed `CurrentVersionURL` in the `CreatePreviewPage` function
- Enables renderer to access the current version URL and redirect users to that version dataset, rather than the most recent dataset

### How to review

- Navigate to a dataset with more than one version - navigate to a non-current version for that dataset.
- Find text, 'Refer to the dataset page for more information...'
- Check that 'dataset page' hyperlink returns to the dataset version you were viewing.

### Who can review

Anyone
